src/plugins/chart_expressions/expression_legacy_metric/common/expression_functions/metric_vis_function.ts

```diff
@@ -45,6 +45,14 @@ export const metricVisFunction = (): MetricVisExpressionFunctionDefinition => ({
     defaultMessage: 'Metric visualization',
   }),
   args: {
+    metricAlignment: {
+      types: ['string'],
+      default: 'center',
+      help: i18n.translate('expressionLegacyMetricVis.function.metricAlignment.help', {
+        defaultMessage: 'Metric alignment',
+      }),
+      required: false,
+    },
     percentageMode: {
```

Comment:
Agree. I've renamed metricAlignment and alignment to `autoScaleMetricAlignment`, so it will be clearer that this is only applied when the metric scales and autoScaleMetricAlignment is given.